tools/http-test-helper/src/main/java/org/enso/shttp/test_helpers/DownloadTestHandler.java

```diff
@@ -0,0 +1,58 @@
+package org.enso.shttp.test_helpers;
+
+import com.sun.net.httpserver.HttpExchange;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.URI;
+import java.util.Random;
+import org.apache.http.client.utils.URIBuilder;
+import org.enso.shttp.SimpleHttpHandler;
+
+/**
+ * A handler that generates a data response, with optional max-age and Age headers. The data
+ * response consists of a string of random letters of the requested length.
+ */
+public class DownloadTestHandler extends SimpleHttpHandler {
+  private Random random = new Random(System.currentTimeMillis());
```

Comment:
Why not set a specific seed to ensure test reproducibility? I usually set the seed to 42.